pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: qr
-version: 1.2.1-dev
+version: 1.2.2-dev
```

Comment:
since this is a new feature, it should be `1.3.0-dev`

Reply:
I'll fixed it.